insights/src/main/kotlin/com/algolia/instantsearch/insights/converter/Converter.kt

```diff
@@ -6,3 +6,12 @@ internal interface Converter<in I, out O> {
     fun convert(input: I): O
     fun convert(inputs: List<I>): List<O> = inputs.map(::convert)
 }
+
+internal const val EventTypeKey = "eventType"
+internal const val EventNameKey = "eventName"
+internal const val IndexNameKey = "index"
+internal const val UserTokenKey = "userToken"
+internal const val TimestampKey = "timestamp"
+internal const val QueryIdKey = "queryId"
+internal const val ObjectIDsKey = "objectIDs"
+internal const val PositionsKey = "positions"
```

Comment:
Enum ? 😢

Reply:
Tried as `Enum`:
```kotlin
enum class Key(val value: String) {
        EventType("eventType"),
        EventName("eventName"),
        IndexName("index"),
        UserToken("userToken"),
        Timestamp("timestamp"),
        QueryId("queryId"),
        ObjectIDs("objectIDs"),
        Positions("positions")
    }
```

But then the API gets ugly with boilerplate:
```kotlin
map[EventNameKey] = eventName // Before 
map[Key.EventName.value] = eventName // After
```

Any idea on how this can be improved?